egs/wsj/s5/steps/libs/tensorflow/kaldi_tensorflow/kaldiInterface.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2016    Vincent Renkens
```

Comment:
please use underscores in names, e.g. kaldi_interface.py.  But I'm not crazy about this name, because it could be confused with some kind of generic python to C++ interface.  Maybe examples.py or example_utils.py or something like that?